src/LooksRare.ts

```diff
@@ -234,6 +236,48 @@ export class LooksRare {
     };
   }
 
+  /**
+   * Create a maker bid for collection offer.
+   * @see this.createMakerBid
+   * @param orderInputs Order data
+   * @returns CreateMakerBidOutput
+   */
+  public createMakerCollectionOffer(orderInputs: CreateMakerCollectionOfferInput): Promise<CreateMakerBidOutput> {
+    return this.createMakerBid({ ...orderInputs, strategyId: StrategyType.collection, itemIds: [] });
+  }
+
+  public async createMakerCollectionOfferWithMerkleTree(
+    orderInputs: CreateMakerCollectionOfferInput,
+    itemIds: BigNumberish[]
+  ): Promise<CreateMakerBidOutput & { proofs: { itemId: BigNumberish; proof: string[] }[] }> {
+    const leaves = itemIds.map((itemId) => {
+      const hash = utils.keccak256(utils.solidityPack(["uint256"], [itemId]));
+      return Buffer.from(hash.slice(2), "hex");
+    });
+    const tree = new MerkleTreeJS(leaves, keccak256, { sortPairs: true });
```

Comment:
I think you might be able to `hashLeaves: true` instead of hashing the leaves by yourself

Reply:
didn't work